code/game/machinery/computer/communications.dm

```diff
@@ -321,9 +321,9 @@
 		return FALSE
 
 	//Should be refactored later, if there's another ship that can appear during a mode with a comm console.
-	if(!istype(loc.loc, /area/almayer/command/cic)) //Has to be in the CIC. Can also be a generic CIC area to communicate, if wanted.
-		to_chat(usr, "<span class='warning'>Unable to establish a connection.</span>")
-		return FALSE
+	//if(!istype(loc.loc, /area/almayer/command/cic)) //Has to be in the CIC. Can also be a generic CIC area to communicate, if wanted.
+	//	to_chat(usr, "<span class='warning'>Unable to establish a connection.</span>")
+	//	return FALSE
```

Comment:
Instead of commenting this or outright removing it, you could create a subtype `communications/cmo`